configcatclient/__init__.py

```diff
@@ -1,31 +1,40 @@
 from .configcatclient import ConfigCatClient
 from .interfaces import ConfigCatClientException
+from .datagovernance import DataGovernance
 
 
-def create_client(sdk_key):
+def create_client(sdk_key, data_governance=DataGovernance.Global):
     """
     Create an instance of ConfigCatClient and setup Auto Poll mode with default options
 
     :param sdk_key: ConfigCat SDK Key to access your configuration.
+    :param data_governance:
```

Comment:
Default: Global. Set this parameter to be in sync with the Data Governance preference on the Dashboard: https://app.configcat.com/organization/data-governance (Only Organization Admins have access)